python_modules/dagster/dagster_tests/core_tests/definitions_tests/test_sensor.py

```diff
@@ -16,13 +13,24 @@ def my_graph():
         pass
 
     sensor = SensorDefinition(evaluation_fn=eval_fn, job=my_graph)
-    assert sensor.jobs[0].name == my_graph.name
+    assert sensor.job.name == my_graph.name
 
     sensor = SensorDefinition(evaluation_fn=eval_fn, pipeline_name="my_pipeline")
     with pytest.raises(
         DagsterInvalidDefinitionError, match="No jobs were provided to SensorDefinition."
     ):
-        sensor.jobs()
+        _ = sensor.job
```

Comment:
nit: the equality is unnecessary. This line can just be `sensor.job`

Reply:
expected pylint to get mad at `sensor.job` but it doesnt!